src/posts/lucky_1.0.0-rc1.cr

```diff
@@ -0,0 +1,239 @@
+class Lucky100rc1Release < BasePost
+  title "Lucky reaches the 1.0 ascent with the first release candidate"
+  slug "lucky-1_0_0-rc1-release"
+
+  def published_at : Time
+    Time.utc(year: 2022, month: 10, day: 1)
+  end
+
+  def summary : String
+    <<-TEXT
+    Lucky 1.0.0-rc1 has been released, and it's a huge milestone
+    for the framework and community.
+    TEXT
+  end
+
+  def content : String
+    <<-MD
+    As many developers know, reaching a 1.0 milestone is huge, and something to
+    be celebrated. We're stoked to release the first prelease of the 1.0 series with
+    `rc1`. We will be using this and the next few release candidates to clean up
+    the rough edges of the framework, and make sure all of the dependencies in the
+    Lucky ecosystem are all solid and ready to take us to the next level.
+
+    We will cover a few notable differences, but you can view all the changes
+    from the [CHANGELOG](https://github.com/luckyframework/lucky/blob/main/CHANGELOG.md)
+
+    ### How to upgrade
+
+    See the [UPGRADE_NOTES](https://github.com/luckyframework/lucky/blob/main/UPGRADE_NOTES.md#upgrading-from-029-to-030).
+
+    You can also view the changes using [LuckyDiff](https://luckydiff.com/?from=0.30.0&to=1.0.0-rc1).
+    This tool helps see what has changed between versions and will be included in upgrade notes from now on.
+
+    ## Here's what's new
+
+    This version releases several breaking changes that were needed in
+    order to make the framework more extensible for the future.
+
+    ### No more DB dependency
+
+    Many sites (including this one) have no need for a databaase. In some cases,
+    your data comes from a 3rd party API, or maybe you need a custom database
+    engine other than PostgreSQL.
+
+    In this release, Avram is no longer a dependency of Lucky, but instead,
+    Lucky is now an optional dependency of Avram. If you're using Avram outside
+    of Lucky, then nothing changes for you. However, for Lucky users using Avram,
+    you'll now have to include it as an additional dependency. Be sure to add it
+    to your `shard.yml` file, and require it in your `src/shards.cr`.
```

Comment:
Does this mean that by default Avram is not included anymore when you use `lucky init`?  Or is this saying that for _existing_ Lucky websites you need to explicitly add it?

If the latter then maybe:

```suggestion
    of Lucky, then nothing changes for you. However, for existing Lucky users upgrading to `rc1` that need Avram,
    you'll now have to include it as an additional dependency. Be sure to add it
    to your `shard.yml` file, and require it in your `src/shards.cr`.
```

Reply:
This is so big. Great work on this!